packages/component-library/stories/Layout.stories.tsx

```diff
@@ -19,7 +19,7 @@ const supportIcon = require("@kaizen/component-library/icons/support.icon.svg")
 import { TitleBlock } from "@kaizen/draft-title-block"
 
 export default {
-  title: "Layout (React)",
+  title: "Layout (React, DEPRECATED)",
```

Comment:
```suggestion
  title: "Layout (deprecated) (React)",
```

just for @didoesdigital 's sake :D

![image](https://user-images.githubusercontent.com/6406263/89268780-57781e00-d67c-11ea-97de-f12d0f96826c.png)

Reply:
Haha, good eye, could I please get a retick?